### PR TITLE
Functional token for ticketing experience on desktop web

### PR DESCRIPTION
**Pull request by karuto**
### Overview
This PR introduces the first "functional token" (issue https://github.com/opentable/design-tokens/issues/91) that's intended to capture all the design decisions about a specific user experience segment / product: ticketing experience on desktop web. The design decisions originated from @francesyun. 😇 

In V1 it will only be used for ticketing modal that I developed. In time, we would like this to be shared across other desktop web ticketing experience products, such as booking flow and rest ref segments. 

The ticketing modal CSS is already 95% design tokens, therefore this token only captures design decisions that are NOT present in other universal tokens. Let me know what you think of this approach.

The variable names are self explanatory enough. I'm very flexible with the naming conventions of the token or the values!

@nickbalestra @matteofigus Let me know what your opinion is on integrating this into the style guide - or whether we should.

Preview:
image

### CODE REVIEW

File: OTKit/otkit-ticketing-desktop/token.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  # Used in ticketing modal of desktop restaurant profile
+  # =============================================
+  modal-max-width:
+    type: "size"
+    value: "672px"
+
+  # Used in ticketing modal of desktop restaurant profile
+  # =============================================
+  modal-background-color:
+    value: "rgba(0,0,0,0.8)"
```
Comment (karuto) [inline OTKit/otkit-ticketing-desktop/token.yml]: Not sure if this should live in `otkit-colors` or not, we should move it to there if this is universal.

File: OTKit/otkit-ticketing-desktop/token.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  # Used in ticketing modal of desktop restaurant profile
+  # =============================================
+  modal-hero-fallback-background-color:
+    value: "#492c6e"
```
Comment (karuto) [inline OTKit/otkit-ticketing-desktop/token.yml]: I don't think @francesyun approved this, but I added a shade of purple that is close to the hamster ball hero image. Feel free to change @francesyun 😭

File: OTKit/otkit-ticketing-desktop/token.yml (hunk 1 above)
Comment (francesyun) [reply OTKit/otkit-ticketing-desktop/token.yml]: this value will be used exclusively for all modal backgrounds. is that enough to move it to otkit-colors?

File: OTKit/otkit-ticketing-desktop/token.yml (hunk 2 above)
Comment (francesyun) [reply OTKit/otkit-ticketing-desktop/token.yml]: haha, this is fine with me :)

File: OTKit/otkit-ticketing-desktop/token.yml (hunk 1 above)
Comment (karuto) [reply OTKit/otkit-ticketing-desktop/token.yml]: yes we should move it to colors, would you like to do it? Or I can do it later :P

Comment (karuto) [reply OTKit/otkit-ticketing-desktop/token.yml]: Either way I am going to remove this from ticketing token for now

**Pull request merged**